fhir-term-graph/src/test/java/com/ibm/fhir/term/graph/test/CodeSystemTermGraphLoaderTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.term.graph.test;
+
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.resource.CodeSystem;
+import com.ibm.fhir.model.resource.CodeSystem.Concept;
+import com.ibm.fhir.term.graph.FHIRTermGraph;
+import com.ibm.fhir.term.graph.factory.FHIRTermGraphFactory;
+import com.ibm.fhir.term.graph.loader.FHIRTermGraphLoader;
+import com.ibm.fhir.term.graph.loader.impl.CodeSystemTermGraphLoader;
+import com.ibm.fhir.term.graph.provider.GraphTermServiceProvider;
+import com.ibm.fhir.term.graph.util.FHIRTermGraphUtil;
+import com.ibm.fhir.term.spi.FHIRTermServiceProvider;
+import com.ibm.fhir.term.util.CodeSystemSupport;
+
+import ch.qos.logback.classic.Level;
+
+public class CodeSystemTermGraphLoaderTest {
+    @Test
+    public void testCodeSystemTermGraphLoader() throws Exception {
+        FHIRTermGraphUtil.setRootLoggerLevel(Level.INFO);
+
+        FHIRTermGraph graph = null;
+        try (InputStream in = CodeSystemTermGraphLoaderTest.class.getClassLoader().getResourceAsStream("JSON/CodeSystem-test.json")) {
+            graph = FHIRTermGraphFactory.open(new PropertiesConfiguration("conf/janusgraph-berkeleyje-lucene.properties"));
```

Comment:
is the graph closeable/ maybe put it in a try  with resources?

Reply:
The graph isn't closeable.

Reply:
Potentially a future refactor